code/src/App.js

```diff
@@ -1,9 +1,24 @@
-import React from 'react'
+/*eslint-disable*/
+import React from 'react';
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import labyrinth from 'reducers/labyrinth';
+import loading from 'reducers/loading';
+import StartScreen from 'components/StartScreen';
+import { Provider } from 'react-redux';
+import SpaceBackground from 'components/SpaceBackground';
 
 export const App = () => {
+  const reducer = combineReducers({
+    labyrinth: labyrinth.reducer,
+    loading: loading.reducer
+  });
+
+const store = configureStore({ reducer });
+
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <Provider store={store}>
+      <SpaceBackground />
+      <StartScreen />
+    </Provider>
   )
```

Comment:
We took notice of the neat organization of the components and keeping the App.js readable and concise.